tools/ci/testlist/sim-03.dat

```diff
@@ -24,6 +24,15 @@
 -Darwin,sim:usbdev
 -Darwin,sim:usbhost
 
+# macOS fails to patch quickjs-libc.c
+-Darwin,sim:quickjs
+
+# macOS fails at uintmax_t: 'unsigned long long' vs 'unsigned long'
+-Darwin,sim:rpproxy_virtio
+
+# macOS fails at uintmax_t: 'unsigned long long' vs 'unsigned long'
+-Darwin,sim:rpserver_virtio
```

Comment:
should we fix this problem instead disabling it?

Reply:
Hmmm can anyone help? Our macOS builds have been failing in our NuttX Mirror for a couple of weeks, kinda distressing to see it failing the NuttX Mirror Build every day 😬

Reply:
could you attach the warning message?

Reply:
I have created the issue:
- https://github.com/apache/nuttx/issues/14775